### PR TITLE
Add timestamp in tap file

### PR DESCRIPTION
**Pull request by renfeiw**
Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
Screen Shot 2022-02-23 at 5 16 33 PM

**Review by sophia-guo — APPROVED** [review]

File: scripts/resultsSum.pl [hunk 1]
```diff
@@ -232,6 +232,8 @@ sub resultReporter {
 			make_path($dir);
 		}
 		open(my $fhOut, '>', $tapFile) or die "Cannot open file $tapFile!";
+		my $timeStamp = localtime;
+		print $fhOut "# Timestamp: " . $timeStamp . "\n";
```
Comment (llxia) [inline scripts/resultsSum.pl:236]: Is this localtime? I think we need to use UTC

**Comment by llxia** [comment]
@renfeiw could you provide the screenshot when you are done? Thanks

**Comment by llxia** [comment]
Thanks @renfeiw . One more thing, could you add `UTC` in the printout, so there is no confusion? For example:

```
print $fhOut "# Timestamp (UTC): " . $timeStamp . "\n";
```

**Review by llxia — APPROVED** [review]
Thanks @renfeiw

**Comment by renfeiw** [comment]
Screen Shot 2022-02-24 at 5 35 30 PM

**Pull request merged**